Makefile

```diff
@@ -131,6 +131,7 @@ update-metadata: build
 .PHONY: doc-setup
 doc-setup: client
 	@echo "Setting up documentation build environment"
+	pip3 install gitpython pyyaml
```

Comment:
Could this be done just after line 136  so its installed in the venv and not globally?

Reply:
Hmm, not sure. It might work ... can't test it locally though because it works on my machine anyway. ;)
Let's try it.